### ISSUE TITLE
Unit Testing in PhantomJS Timeout

### ISSUE THREAD

**Issue by gonzofish** [angular/material2#2921]
#### Bug, feature request, or proposal:
When unit testing, in PhantomJS 2.1.1 (not sure about other versions), if I import `MaterialModule.forRoot()`, the tests time out. If I pull in the modules for only the Material2 components that are used in my component, the test succeeds.

Note, this does not happen when testing through Chrome.

#### What is the expected behavior?
That the tests do not fail because of an async timeout

#### What is the current behavior?
PhantomJS (or Jasmine) spits out the following message:

```shell
Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
```

#### What are the steps to reproduce?

Plnkr: http://plnkr.co/edit/qw1OLZ

Basically just create a component with an Angular material component in it and import `MaterialModule.forRoot()` into your test for that component.

#### What is the use-case or motivation for changing an existing behavior?

I want to effectively test my components which use Material2

#### Which versions of Angular, Material, OS, browsers are affected?
Angular 2.3.1
Material2 2.0.0-beta.1
PhantomJS 2.1.1
Windows 10
Node  4.5.0

#### Is there anything else we should know?

No

**Comment by DevVersion** [comment]
Hey @gonzofish - I tried to reproduce this locally, but didn't saw the issue.


![](https://i.gyazo.com/bcb4fb77672145c066e9fa7dacd94e26.png)

Please try to update your PhantomJS / launcher and also try to use Angular Material beta.2

**Comment by mohsenvafa** [comment]
I have the same issue when i run tests with phantom js. I get this error only for components that uses material component.
I am using beta.8 with most updated PhantomJS

![image](https://user-images.githubusercontent.com/5973501/29592578-e01a1bb6-8759-11e7-998b-b742929b62f6.png)

**Comment by jimmyshephard** [comment]
Any fix or work around for this? Getting the same error on a component test
